data/json/items/comestibles.json

```diff
@@ -5624,7 +5624,7 @@
     {
         "type" : "COMESTIBLE",
         "id" : "marloss_seed",
-        "name" : "marloss seed",
+        "name" : "marloss seeds",
         "name_plural" : "marloss seeds",
```

Comment:
As the marloss seed is described as ''This looks like a sunflower seed the size of your palm." perhaps it is an idea to keep this one singular? Esp as this is an alien type of seed.

Reply:
Fair enough.